python_src/src/lamp_py/performance_manager/l0_gtfs_static_mod.py

```diff
@@ -0,0 +1,184 @@
+import sqlalchemy as sa
+
+
+from lamp_py.postgres.postgres_utils import DatabaseManager
+from lamp_py.postgres.postgres_schema import (
+    StaticStopTimes,
+    StaticStops,
+    StaticTrips,
+)
+from lamp_py.runtime_utils.process_logger import ProcessLogger
+
+
+def generate_scheduled_travel_times(
+    fk_timestamp: int, db_manager: DatabaseManager
+) -> None:
+    """
+    generate scheduled travel_times and insert into static_stop_times table
+    """
+    travel_times_cte = (
+        sa.select(
+            StaticStopTimes.pk_id,
+            (
+                StaticStopTimes.arrival_time
+                - sa.func.lag(StaticStopTimes.departure_time).over(
+                    partition_by=StaticStopTimes.trip_id,
+                    order_by=StaticStopTimes.stop_sequence,
+                )
+            ).label("schedule_travel_time_seconds"),
+        )
+        .select_from(StaticStopTimes)
+        .where(
+            StaticStopTimes.timestamp == fk_timestamp,
+        )
+        .cte("scheduled_travel_times")
+    )
```

Comment:
These calls are all very similar, using window functions to calculate metrics and the performing and `UPDATE FROM` call on the RDS to update the columns.